### PR TITLE
[I18N] [16.0] calendar: update translation file (.pot)

### PR DESCRIPTION
**Pull request by vietphuong97**
update translation file

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by vietphuong-viindoo** [comment]
Hello @arj-odoo , can you review this PR for me? It simply updates the translation file (.pot)

File: addons/calendar/i18n/calendar.pot [hunk 1]
```diff
@@ -1167,13 +1167,6 @@ msgstr ""
 msgid "Group By"
 msgstr ""
 
-#. module: calendar
-#. odoo-python
-#: code:addons/calendar/models/calendar_event.py:0
-#, python-format
-msgid "Grouping by %s is not allowed on private events."
-msgstr ""
-
```
Comment (arj-odoo) [inline addons/calendar/i18n/calendar.pot:1176]: We don't delete terms in old versions.

File: addons/calendar/i18n/calendar.pot [hunk 2]
```diff
@@ -1658,13 +1651,6 @@ msgstr ""
 msgid "Private"
 msgstr ""
 
-#. module: calendar
-#. odoo-python
-#: code:addons/calendar/models/calendar_event.py:0
-#, python-format
-msgid "Private Event Excluded"
-msgstr ""
-
```
Comment (arj-odoo) [inline addons/calendar/i18n/calendar.pot:1667]: Same

File: addons/calendar/i18n/calendar.pot [hunk 3]
```diff
@@ -397,7 +397,7 @@ msgid ""
 "                <t t-out='format_datetime(dt=object.event_id.start, tz=object.mail_tz if not object.event_id.allday else None, dt_format=\"EEEE\", lang_code=object.env.lang) or \"\"'>Tuesday</t>\n"
 "            </div>\n"
 "            <div style=\"font-size: 48px; min-height: auto; font-weight: bold; text-align: center; color: #5F5F5F; background-color: #F8F8F8; border: 1px solid #875A7B;\">\n"
-"                <t t-out=\"str(object.event_id.start.day) or ''\">4</t>\n"
+"                <t t-out=\"format_datetime(dt=object.event_id.start, tz=object.mail_tz if not object.event_id.allday else None, dt_format='d', lang_code=object.env.lang) or ''\">4</t>\n"
 "            </div>\n"
```
Comment (arj-odoo) [inline addons/calendar/i18n/calendar.pot:401]: Are these changes done in xml but not propagated in the pot file?

I find it strange that you update pot file and not the related xml in this PR. Note that the related xml won't be automatically updated.

File: addons/calendar/i18n/calendar.pot [hunk 4]
```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 16.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-05-23 08:22+0000\n"
-"PO-Revision-Date: 2023-05-23 08:22+0000\n"
+"POT-Creation-Date: 2023-10-21 07:18+0000\n"
+"PO-Revision-Date: 2023-10-21 07:18+0000\n"
```
Comment (arj-odoo) [inline addons/calendar/i18n/calendar.pot:10]: This is considered useless diff.

**Review by arj-odoo — CHANGES_REQUESTED** [review]
I have a few question and some of your changes are not useful and should be discarded.

**Pull request closed without merging**